### PR TITLE
style: Manual lint fixes

### PR DESCRIPTION
**Pull request by tuliomir**
### Acceptance Criteria
- Fix all lint warnings and errors on this application
- The only exception is the parsing error found on most screens: that will only be solved by migrating them to functional components

### Security Checklist
- [X] Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.

### CODE REVIEW

File: src/api/addressApi.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ const addressApi = {
       if (res && res.data) {
         return res.data;
       }
+      return undefined;
```
Comment (r4mmer) [inline src/api/addressApi.js:30]:
```suggestion
      return;
```

Would this also work?

File: src/api/customAxiosInstance.js [hunk 2]
```diff
@@ -17,9 +17,9 @@ import hathorLib from '@hathor/wallet-lib';
  * Create an axios instance to be used when sending requests
  *
  * @param {callback} resolve Callback to be stored and used in case of a retry after a fail
- * @param {number} timeout Timeout in milliseconds for the request
+ * @param {number} [_timeout] Timeout in milliseconds for the request
  */
-const createRequestInstance = (resolve, timeout) => {
+const createRequestInstance = (resolve, _timeout) => {
```
Comment (r4mmer) [inline src/api/customAxiosInstance.js:22]: suggestion(if-minor): Should we remove timeout?

If its not used we could just remove this argument, right?

**Review by r4mmer — APPROVED** [review]

File: src/api/addressApi.js (hunk 1 above)
Comment (tuliomir) [reply src/api/addressApi.js:30]: It wouldn't: removing the explicit `undefined` raises an error about an unnecessary `return` on the last line of the function, and removing the `return` breaks the `consistent-return` lint rule that I intended to fix in the first time.

Maybe we can find a better solution at another time, but for now I'd rather not play with code logic.

File: src/api/customAxiosInstance.js (hunk 2 above)
Comment (tuliomir) [reply src/api/customAxiosInstance.js]: Good point. Removed on https://github.com/HathorNetwork/hathor-explorer/pull/294/commits/f38cf8ecff200cf9b0fb4764fc167158f13ac1e7

**Review by r4mmer — APPROVED** [review]

File: src/api/metadataApi.js [hunk 3]
```diff
@@ -15,8 +15,9 @@ const metadataApi = {
         if (res && id in res.data) {
           return res.data[id];
         }
+        return undefined;
       })
-      .catch(error => {
+      .catch(_error => {
```
Comment (andreabadesso) [inline src/api/metadataApi.js:20]: can't you do:

```suggestion
      .catch(() => {
```

?

**Review by andreabadesso — APPROVED** [review]

File: src/api/metadataApi.js (hunk 3 above)
Comment (tuliomir) [reply src/api/metadataApi.js:20]: I'd rather keep the variable, so that we feel more inclined to properly handle these errors in the future

**Pull request merged**